.github/actions/detect-workflow/action.yml

```diff
@@ -21,6 +21,9 @@ outputs:
   ref:
     description: The current workflow reference
     value: ${{ steps.detect.outputs.ref }}
+  workflow:
```

Comment:
note for review: i've added this, because i'd like to consolidate the number of places that we have to fetch an OIDC token in order to detect the reusable workflow. this will help if GitHub ever exposes this in a workflow context to remove the restrictions on `pull_request`.

(In the delegator, we use the TRW as the builder.id, which we get from the SLSA token.)

Reply:
in the delegator, we don't even call detect-env, correct?

Reply:
HMM! You're right - we populate the builder ID via the tool's slsa-token, so we never need to introspect. Perfect.